integration-test/.jshintrc

```diff
@@ -0,0 +1,3 @@
+{
+	"mocha": true
+}
```

Comment:
nit, please try to have newlines at end of file if possible (Maybe we can add this to the lint check)

Reply:
There are other files in this PR with that issue.

Reply:
> 4 lines (3 sloc)